simplecpp.cpp

```diff
@@ -457,7 +457,7 @@ static bool isStringLiteralPrefix(const std::string &str)
            str == "R" || str == "uR" || str == "UR" || str == "LR" || str == "u8R";
 }
 
-void simplecpp::TokenList::lineDirective(unsigned int fileIndex, unsigned int line, Location *location)
+void simplecpp::TokenList::lineDirective(std::size_t fileIndex, unsigned int line, Location *location)
```

Comment:
it's rather excessive to use a 64-bit value for fileIndex. you might see 10-20 files in a translation unit but hardly millions of files..

Reply:
Yes, you're right.  For some reason, I thought that `fileIndex` was a file offset.